presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PreferredProperties.java

```diff
@@ -430,5 +413,28 @@ public String toString()
                     .add("nullsAndAnyReplicated", nullsAndAnyReplicated)
                     .toString();
         }
+
+        private PartitioningProperties mergeWithParent(PartitioningProperties parent)
```

Comment:
Why to move this method?

Reply:
Same as before, made private and then moved below the public methods.